src/System.ComponentModel.Composition/ref/System.ComponentModel.Composition.cs

```diff
@@ -7,6 +7,12 @@
 
 namespace System.ComponentModel.Composition
 {
+    public static partial class AdaptationConstants
```

Comment:
@maryamariyan when I regenerated the ref for System.ComponentModel.Composition  a few more public API showed up. Can you confirm you didn't intentionally exclude them originally.?